### PR TITLE
[Issue 3363] Extension -  Update connector version 

### PR DESCRIPTION
**Pull request by Thiendekaco**

### CODE REVIEW

**Comment by saltict** [comment]
🚀 Deployed on https://pr-3437--sw-web-runner.netlify.app

File: packages/extension-base/src/services/wallet-connect-service/index.ts [hunk 1]
```diff
@@ -176,10 +176,29 @@ export default class WalletConnectService {
     }
   }
 
+  async #onPingReply ({ topic }: SignClientTypes.EventArguments['session_ping']) {
+    if (!(await this.haveData())) {
```
Comment (S2kael) [inline packages/extension-base/src/services/wallet-connect-service/index.ts:180]: Why need?

File: packages/extension-base/src/services/wallet-connect-service/index.ts [hunk 2]
```diff
@@ -176,10 +176,29 @@ export default class WalletConnectService {
...
+      await this.initClient(true);
+    }
+
+    this.#checkClient();
+
+    try {
+      const requestSession = this.getSession(topic);
+      const sessionAccounts = requestSession.namespaces.eip155.accounts.map((account) => account.split(':')[2]);
```
Comment (S2kael) [inline packages/extension-base/src/services/wallet-connect-service/index.ts:188]: Substrate cannot use this event?

File: packages/extension-base/src/koni/background/handlers/Tabs.ts [hunk 3]
```diff
@@ -451,6 +457,38 @@ export default class KoniTabs {
     }];
   }
 
+  private async revokePermissions (url: string, id: string, { params }: RequestArguments) {
```
Comment (S2kael) [inline packages/extension-base/src/koni/background/handlers/Tabs.ts:460]: Defined list permission can revoke

File: packages/extension-base/src/koni/background/handlers/Tabs.ts [hunk 4]
```diff
@@ -451,6 +457,38 @@ export default class KoniTabs {
...
+    const permissions = Object.keys(params[0] as Record<string, any>);
+
+    const permissionPromise = async (permission: string): Promise<void> => {
+      if (permission === 'eth_accounts') {
+        return new Promise((resolve) => {
+          this.#koniState.getAuthorize((value) => {
+            const urlStripped = stripUrl(url);
+
+            if (value && value[urlStripped]) {
+              delete value[urlStripped];
```
Comment (S2kael) [inline packages/extension-base/src/koni/background/handlers/Tabs.ts:474]: Handle case domain have both evm and substrate authorized permission

File: packages/extension-base/src/koni/background/handlers/Tabs.ts [hunk 5]
```diff
@@ -451,6 +457,38 @@ export default class KoniTabs {
     }];
   }
 
+  private async revokePermissions (url: string, id: string, { params }: RequestArguments) {
+    if (!params || !isArray(params) || params.length === 0) {
+      throw new EvmProviderError(EvmProviderErrorType.INVALID_PARAMS, 'No list of permissions found to revoke in the parameters.');
+    }
+
+    const permissions = Object.keys(params[0] as Record<string, any>);
```
Comment (S2kael) [inline packages/extension-base/src/koni/background/handlers/Tabs.ts:465]: Standardize data, filter duplicates

File: packages/extension-base/src/koni/background/handlers/Tabs.ts [hunk 6]
```diff
@@ -362,7 +369,6 @@ export default class KoniTabs {
             const result = accountList.filter((adr) => adr !== address);
 
             result.unshift(address);
```
Comment (S2kael) [inline packages/extension-base/src/koni/background/handlers/Tabs.ts:371]: Why remove this line?

**Review by S2kael — CHANGES_REQUESTED** [review]

File: packages/extension-base/src/koni/background/handlers/Tabs.ts [hunk 7]
```diff
@@ -451,6 +451,79 @@ export default class KoniTabs {
...
+                    if (isEthereumAddress(address)) {
+                      allowedMap[address] = false;
+                    } else {
+                      allowedMap[address] = value;
+                    }
+
+                    return allowedMap;
+                  }, {});
+
+                  break;
```
Comment (S2kael) [inline packages/extension-base/src/koni/background/handlers/Tabs.ts:507]: missing change type

File: packages/extension-base/src/koni/background/handlers/Tabs.ts [hunk 8]
```diff
@@ -451,6 +451,79 @@ export default class KoniTabs {
...
+  private async revokePermissions (url: string, id: string, { params }: RequestArguments) {
+    const revokePermissions: Record<string, boolean> = {
+      eth_accounts: true
+    };
+
+    if (!params || !isArray(params) || params.length === 0) {
+      throw new EvmProviderError(EvmProviderErrorType.INVALID_PARAMS, 'No list of permissions found to revoke in the parameters.');
+    }
+
+    const permissions = Object.keys(params[0] as Record<string, any>).filter((permission) => {
```
Comment (S2kael) [inline packages/extension-base/src/koni/background/handlers/Tabs.ts:463]: Simpler with `Set`

File: packages/extension-base/src/koni/background/handlers/Tabs.ts [hunk 9]
```diff
@@ -451,6 +451,79 @@ export default class KoniTabs {
     }];
   }
 
+  private async revokePermissions (url: string, id: string, { params }: RequestArguments) {
+    const revokePermissions: Record<string, boolean> = {
```
Comment (S2kael) [inline packages/extension-base/src/koni/background/handlers/Tabs.ts:455]: Is it necessary to have a default value?

**Review by S2kael — CHANGES_REQUESTED** [review]

**Review by S2kael — APPROVED** [review]

**Review by S2kael — APPROVED** [review]

**Pull request merged**